lib/pages/saved_places/saved_places.dart

```diff
@@ -0,0 +1,155 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:latlong/latlong.dart';
+
+import 'package:trufi_core/blocs/search_locations/search_locations_cubit.dart';
+import 'package:trufi_core/l10n/trufi_localization.dart';
+import 'package:trufi_core/pages/saved_places/location_tiler.dart';
+import 'package:trufi_core/widgets/dialog_edit_text.dart';
+
+import '../../trufi_models.dart';
+import '../../widgets/trufi_drawer.dart';
+import '../choose_location.dart';
+
+class SavedPlacesPage extends StatelessWidget {
+  static const String route = '/places';
+
+  const SavedPlacesPage({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final searchLocationsCubit = context.read<SearchLocationsCubit>();
+    final localization = TrufiLocalization.of(context);
+    final theme = Theme.of(context);
+    return Scaffold(
+      appBar: AppBar(title: Text(localization.menuYourPlaces)),
+      body: DefaultTabController(
+        length: 2,
+        child: Column(
+          children: [
+            const SizedBox(height: 10),
+            TabBar(
+              tabs: [
+                Container(
+                  padding: const EdgeInsets.all(10),
+                  child: Text(
+                    // TODO translate
+                    "Saved places",
+                    style: theme.textTheme.bodyText1,
+                  ),
+                ),
+                Container(
+                  padding: const EdgeInsets.all(10),
+                  child: Text(
+                    // TODO translate
+                    "Favorite places",
+                    style: theme.textTheme.bodyText1,
+                    maxLines: 2,
+                  ),
+                )
+              ],
+            ),
+            const SizedBox(height: 5),
+            Expanded(
+              child: TabBarView(
+                children: [
+                  Scaffold(
```

Comment:
Flutter Documentation
```
/// Although there are some use cases, like a presentation app that
/// shows embedded flutter content, where nested scaffolds are
/// appropriate, it's best to avoid nesting scaffolds.
```

In our case, we need a FloatingActionButton if we are on the first screen is it really mandatory to create a second `Scaffold`?

Reply:
I used the second scaffold for the fast implementation because I think there will be a change in the UI, I understand your suggestion, and I will use a stack. 👍